src/AppBundle/Resources/views/book/edit.html.twig

```diff
@@ -19,6 +19,7 @@
 
         <div class='btn-toolbar pull-right'>
             <div class='btn-group'>
+                <a href="{{ path('book_show_contributions', { 'id': book.id }) }}" class="btn btn-primary">Edit contributions</a>
```

Comment:
I think you could replace "Edit" with an edit icon. Maybe `<span class="glyphicon glyphicon-edit"></span>` to give it a bit more consistency.